apps/src/dance/lab2/views/GenerateDance.tsx

```diff
@@ -117,8 +122,8 @@ const GenerateDance: React.FunctionComponent<GenerateCodeProps> = ({
     setAiGenerateState('generating');
 
     const startTime = Date.now();
-    const resultBlockly = buildDanceBlockly(
-      measures,
+    const {workspaceSerialization, flyoutDefinition} = buildDanceBlockly(
+      measures.filter(measure => measure !== 1), // First measure is redundant with "when run"
```

Comment:
optional: can we just omit measure 1 in `buildDanceBlockly`?

Reply:
Yes, that's true. Either way probably works. It probably makes sense for that particular context to be known/understood best by `buildDanceBlockly`, so I'll move it.